engine/src/main/java/org/terasology/config/RenderingConfig.java

```diff
@@ -491,10 +528,10 @@ public boolean isVSync() {
         return this.vSync;
     }
 
-    public static final String V_SYNC = "VSync";
-    public void setVSync(boolean vSync) {
-        this.vSync = vSync;
-        propertyChangeSupport.firePropertyChange(V_SYNC, !this.vSync, this.vSync);
+    public void setVSync(boolean vsync) {
```

Comment:
I'm not really sure here why vsync was flagged up as hiding a field while the method above (and others), using the input parameter "meshLimit", doesn't.